.github/workflows/branch--lint-unit-and-smoke-test.yml

```diff
@@ -113,6 +113,8 @@ jobs:
         run: docker-compose up -d
       - name: Install smoke tests dependencies
         run: yarn workspaces focus @xgovformbuilder/test
+      - name: Install pa11y
+        run: npm install -g pa11y-ci
```

Comment:
Are you expecting to only run this in the designer, on the CI? the equivalent command for npm's "npx" is "dlx" for yarn. So, you can just do this `yarn {{matrix.app}} dlx pa11y-ci`